workers/generic-worker/fileutil/fileutil_windows.go

```diff
@@ -16,3 +19,15 @@ func SecureFiles(filepaths ...string) (err error) {
 	}
 	return
 }
+
+func ResetPermissions(path string, permissions fs.FileMode) error {
```

Comment:
It looks like permissions isn't used here. What does reset mean in this context? How does Windows know what the previous permissions were? Is it removing all permissions associated with the path, or applying some previous value?

Reply:
I had to make the function signature the same but have two different implementations between posix and windows. That's why `permissions` isn't used here, where it is on posix.

And `/reset`'s docs on https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/icacls says:

>`[/reset [/t] [/c] [/l] [/q]] 	Replaces ACLs with default inherited ACLs for all matching files.`